### ISSUE TITLE
Nitro fails to show version, updates – but appears to still work?

### ISSUE THREAD

**Issue by OneMohrTime** [craftcms/nitro#270]
### Description

I followed the installation instructions, and completely removed multipass from my Mac. I installed through `homebrew`, and when I checked the version number, I got:
```bash
nitro version
Error: unable to get version from the gRPC API
```

Naturally, I just tried to re-update, but I'm up-to-date
```bash
nitro self-update
Checking for updates
up to date!

nitro update
Error: there is no config file for the environment
```

I'm not sure what config file its referencing, as there isn't one mentioned in the [instructions](https://craftcms.com/docs/nitro/2.x/updating.html#macos-linux-and-windows-self-update). `nitro help` also outputs all of the options, as expected – so _some_ commands work.

Is this just a bug?

![Screen Shot 2021-03-03 at 17 06 25](https://user-images.githubusercontent.com/11075805/109879755-021ce300-7c44-11eb-8609-761bcd2cd9ff.png)

### Steps to reproduce

1. [Uninstall Nitro 1](https://craftcms.com/docs/nitro/1.x/installation.html#uninstalling-nitro)
2. [Install Nitro 2](https://craftcms.com/docs/nitro/2.x/installation.html)

### Additional info

- Nitro version: 2.0.3
- Docker version: 3.1.0

---

### EDIT

Solved by `nitro init`. Just needs to be run once, or else of course there's no version.

**Comment by jasonmccallister** [comment]
@OneMohrTime the `version` command is used to get the version of Docker, Nitro API (gRPC), and the CLI version. 

From the error message above, it appears you did not run `nitro init` which is why it cannot talk to the API that does not exist.

To get the CLI version from Nitro, you can run `nitro -v`. 

Perhaps we should update the `version` command to suggest running `start` or `init` to make it more user friendly.

**Comment by OneMohrTime** [comment]
Totally got it after that. I wasn't paying full attention during the switchover, I should have read more closely 🤦

**Comment by jasonmccallister** [comment]
@OneMohrTime the next release will check if the `init` command has been run and prompt the user to run `init` before continuing. 
 
![CleanShot 2021-03-04 at 10 20 27@2x](https://user-images.githubusercontent.com/5354908/109986113-76529780-7cd3-11eb-91c2-8bbaace2358f.png)

**Comment by OneMohrTime** [comment]
Great solution!

**Comment by jasonmccallister** [comment]
This was released with 2.0.4!

**Comment by craftedsystems** [comment]
I still get this error `Error: unable to get version from the gRPC API` with nitro 2.0.8.
